packages/ddp-server/stream_server.js

```diff
@@ -14,12 +15,18 @@ import once from 'lodash.once';
 var websocketExtensions = once(function () {
   var extensions = [];
 
-  var websocketCompressionConfig = process.env.SERVER_WEBSOCKET_COMPRESSION
-        ? JSON.parse(process.env.SERVER_WEBSOCKET_COMPRESSION) : {};
+  var websocketCompressionConfig = process.env.SERVER_WEBSOCKET_COMPRESSION ?
+    JSON.parse(process.env.SERVER_WEBSOCKET_COMPRESSION) : {};
+
   if (websocketCompressionConfig) {
-    extensions.push(Npm.require('permessage-deflate').configure(
-      websocketCompressionConfig
-    ));
+    extensions.push(Npm.require('permessage-deflate2').configure({
+      threshold: 512,
+      level: zlib.constants.Z_BEST_SPEED,
+      memLevel: zlib.constants.Z_MIN_MEMLEVEL,
+      noContextTakeover: true,
+      maxWindowBits: zlib.constants.Z_MIN_WINDOWBITS,
+      ...(websocketCompressionConfig || {})
+    }));
   }
```

Comment:
Are those configs documented? this whole config could become even a guide for increasing performance

Reply:
I don't think there are docs for it, I can add them

Reply:
Added them https://github.com/meteor/meteor/pull/13445/commits/c73515470f5a4a623b8403231fbfa6c8472cd5b1